src/main/java/utilities/ResponseHandler.java

```diff
@@ -0,0 +1,187 @@
+package utilities;
+
+import tasks.Task;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * A utility class for returning responses as a String.
+ */
+public class ResponseHandler {
+    /**
+     * The message to be printed or stored.
+     */
+    public String message;
+
+    /**
+     * Constructs a ResponseHandler with the given message.
+     *
+     * @param message The message to be printed or stored.
+     */
+    public ResponseHandler(String message) {
+        this.message = message;
+    }
+
+    /**
+     * Prints the stored message to the console.
+     *
+     * @return The stored message.
+     */
+    public String printStored() {
+        return this.message;
+    }
+
+    /**
+     * Generates a personalized reply based on the given message.
+     *
+     * @param message The message to be analyzed and replied to.
+     * @return A personalized reply.
+     */
+    public static String personalisedReply(String message) {
+        HashMap<String, String> responseMap = new HashMap<>();
```

Comment:
I like your usage of a hashmap, I didn't think of it. It definitely makes the run time faster should there be a lot of responses that need to be stored.